.github/workflows/test.yml

```diff
@@ -0,0 +1,69 @@
+name: Test
+
+on:
+  - push
+  - pull_request
+
+env:
+  CI: "true"
+  FORCE_COLOR: "true"
+  DEBUG: "*"
+
+jobs:
+  test:
+    name: Test Node v${{ matrix.node }} on ${{ matrix.os }} and ${{ matrix.puppeteer_version }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        puppeteer_version:
+          # - 5.0.0
+          - 2.1.1 # Chromium 79.0.3942.0, Oct 24 2019
```

Comment:
From my perspective testing the two latest major Puppeteer releases should be fine (currently: 5.0.0 and 2.1.1).

Historically I've added more and more versions as they appeared (being somewhat proud of our backwards compatibility) but that made waiting for the CI results quite annoying. 😄

Reply:
Yeah, I'll enabled the 5 back when I am done fixing build / test steps. I just didn't want to run the entire matrix for fixing Windows-related issues only. I just commented out items in the matrix to only run one Windows test.

Actions do run in parallel, but there does seem to be some limit as to how many. Might be dependent on availability of boxes too. We can test different number of items to see how long it takes.

Reply:
Full matrix run now (without Windows):

https://github.com/berstend/puppeteer-extra/actions/runs/164138527